test/Tester/StreamingTests/EHImplicitSubscriptionStreamRecoveryTests.cs

```diff
@@ -117,7 +75,56 @@ public EHImplicitSubscriptionStreamRecoveryTests()
             // send end events
             for (int j = 0; j < streamCount; j++)
             {
-                await producers[j].OnNextAsync(new GeneratedEvent { EventType = GeneratedEvent.GeneratedEventType.End });
+                await producers[j].OnNextAsync(new GeneratedEvent { EventType = GeneratedEvent.GeneratedEventType.Report });
+            }
+        }
+
+        private class Fixture : BaseClusterFixture
```

Comment:
I think making the fixture a nested class is the right approach (I think I fixed a few of those already). Nevertheless adding it at the bottom is probably not, as this context is important before reading the tests themselves (a rule in TDD is to not have magic setting up stuff, but if you do, make it very explicit).

Having said that, whether you agree or not, since this set up code was already on the top, moving it here makes diffing more painful.

Same comment applies to fixtures in other tests that were moved in this PR.